packages/node_modules/@webex/plugin-meetings/src/common/errors/webex-meetings-error.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @extends Error
+ * @property {number} code - Error code
+ */
+export default class WebexMeetingsError extends Error {
+  /**
+   * Creates a new {@link WebexMeetingsError}
+   * @param {number} code - Error code
+   * @param {string} [message] - Error message
+   * @param {string} [fileName] - Name of the script file where error was generated
+   * @param {number} [lineNumber] - Line number of the script file where error was generated
+   */
+  constructor(code, ...args) {
+    super(...args);
+
+    this.name = 'WebexMeetingsError';
+
+    Object.defineProperty(this, 'code', {
```

Comment:
Is there a reason why we aren't just doing `this.code = {`?

Reply:
I'm fine with the use of `Object.defineProperty` then just trying to reassign the property directly

Reply:
hmmm i feel its becaure to set  enumerator value